src/monster.cpp

```diff
@@ -1216,6 +1216,21 @@ int monster::hit_roll() const {
     return dice(type->melee_skill, 10);
 }
 
+bool monster::has_grab_break_tec() const
+{
+    return false;
+}
+
+bool monster::stability_check( monster *z, int strength) const
```

Comment:
It shouldn't take a monster argument, but just a difficulty argument. Compare with melee attack rolls: they supply a ready attack roll.

That way stability check could be used against explosions and player techniques.